pyrelational/informativeness/regression.py

```diff
@@ -172,7 +172,7 @@ def _check_regression_informativeness_input(
         assert 1 <= std.ndim <= 2, "std input should be a 1D or 2D tensor"
 
 
-def _compute_mean(x: Optional[Union[Distribution, Tensor]], axis: int = 0) -> Tensor:
+def _compute_mean(x: Optional[Union[Distribution, Tensor]], axis: int = 0) -> Union[Tensor, Any]:
```

Comment:
hmmm, I don't think we should have Any output here, it should always be a Tensor (or nothing when raising)